src/main/java/duke/task/Deadlines.java

```diff
@@ -0,0 +1,37 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadlines extends Task {
+    protected LocalDate by;
+
+    public Deadlines(String description, LocalDate by) {
+        super(description);
+        this.by = by;
+    }
+
+    public static Deadlines load(String taskDetails) {
+        String[] splitTaskDetails = taskDetails.split("\\|",4);
+        for (int i = 0; i < splitTaskDetails.length; i++) {
+            splitTaskDetails[i] = splitTaskDetails[i].strip();
+        }
+        Deadlines deadline = new Deadlines(splitTaskDetails[2],
+                LocalDate.parse(splitTaskDetails[3]));
```

Comment:
I'm not too sure about this, but should the statement on line 20 be indented 8 extra spaces here?

(really not sure, the style guide is super vague)